data/partials/footer.ja.yaml

```diff
@@ -16,6 +16,9 @@ col1:
   - name: apm
     link: 'https://www.datadoghq.com/product/apm/'
     title: APM
+  - name: profiler
+    link: "https://www.datadoghq.com/product/code-profiling/"
```

Comment:
Hmmmm, maybe all these links to product pages should be pointing to /ja/product/ instead of English? And same with the /fr/?

Reply:
updated for /ja